css/ood_styles.css

```diff
@@ -168,8 +167,7 @@ span.owner, span.mode {
 
 .panel-btns {
     position: absolute;
-    top: -40px;
-    right: 0px;
+    right: 30px;
```

Comment:
adding `top: 10px` makes it look nice I think - tiny space above (only tested in chrome though)